tests/test_download.py

```diff
@@ -98,13 +99,13 @@ def test_fetch_workflow_details_no_search_result(self, mock_workflows):
         download_obj.fetch_workflow_details(mock_workflows)
 
     #
-    # Tests for 'download_wf_files'  
+    # Tests for 'download_wf_files'
     #
     def test_download_wf_files(self):
         download_obj = DownloadWorkflow(
             pipeline = "dummy",
             release = "1.2.0",
-            outdir="/tmp"
+            outdir = tempfile.mkdtemp()
```

Comment:
Sure :+1: I actually changed it to this because I don’t think it was cleaning up before, so when I ran the tests more than once they failed. But yes, it’d probably be good to delete the temp dirs at the end of each test block..

Reply:
> I actually changed it to this because I don’t think it was cleaning up before

Yes, you are right. And because of the fixed path, it gave a OSError. It is better like this, creating a hashed dirname :+1:

Reply:
Ok, updated 👍